assets/wizards/popups/components/segment-group/SegmentGroup.test.js

```diff
@@ -0,0 +1,462 @@
+				frequency: 'always',
+			},
+			status: 'publish',
+			title: 'Above Header Prompt 3 with category',
+		},
+		{
+			campaign_groups: [],
+			categories: [
+				{
+					name: 'Tech',
+					term_id: 1,
+				},
+				{
+					name: 'News',
+					term_id: 2,
+				},
+			],
+			content: 'Above Header Prompt 4 with category',
+			id: 8,
+			options: {
+				placement: 'above_header',
+				selected_segment_id: SEGMENT.id,
+				frequency: 'always',
+			},
+			status: 'publish',
+			title: 'Above Header Prompt 4 with category',
+		},
+	],
+	customPlacementsUncategorized: [
+		{
+			campaign_groups: [],
+			categories: [],
+			content: 'Custom Placement Prompt 1',
+			id: 9,
+			options: {
+				placement: 'custom1',
+				selected_segment_id: SEGMENT.id,
+				frequency: 'always',
+			},
+			status: 'publish',
+			title: 'Custom Placement Prompt 1',
+		},
+		{
+			campaign_groups: [],
+			categories: [],
+			content: 'Custom Placement Prompt 2',
+			id: 10,
+			options: {
+				placement: 'custom1',
+				selected_segment_id: SEGMENT.id,
+				frequency: 'always',
+			},
+			status: 'publish',
+			title: 'Custom Placement Prompt 2',
+		},
+	],
+	customPlacementsCategorized: [
+		{
+			campaign_groups: [],
+			categories: [
+				{
+					name: 'Tech',
+					term_id: 1,
+				},
+			],
+			content: 'Custom Placement Prompt 3 with category',
+			id: 11,
+			options: {
+				placement: 'custom1',
+				selected_segment_id: SEGMENT.id,
+				frequency: 'always',
+			},
+			status: 'publish',
+			title: 'Custom Placement Prompt 3 with category',
+		},
+		{
+			campaign_groups: [],
+			categories: [
+				{
+					name: 'Tech',
+					term_id: 1,
+				},
+				{
+					name: 'News',
+					term_id: 2,
+				},
+			],
+			content: 'Custom Placement Prompt 4 with category',
+			id: 12,
+			options: {
+				placement: 'custom1',
+				selected_segment_id: SEGMENT.id,
+				frequency: 'always',
+			},
+			status: 'publish',
+			title: 'Custom Placement Prompt 4 with category',
+		},
+	],
+};
+
+describe( 'A segment with conflicting prompts', () => {
+	beforeEach( () => {
+		// Mock global vars for custom placements.
+		window.newspack_popups_wizard_data = {
+			custom_placements: {
+				custom1: 'Custom Placement 1',
+			},
+		};
+	} );
+
+	it( 'renders a conflict notice for uncategorized overlays in the same segment', async () => {
+		SEGMENT.prompts = PROMPTS.overlaysUncategorized;
+
+		// Expected warning text.
+		const noticeText =
+			'If multiple uncategorized overlays share the same segment, only the most recent one will be displayed.';
+		const props = {
+			campaignData: CAMPAIGN.campaignData,
+			campaign: CAMPAIGN.campaignId,
+			segment: SEGMENT,
+		};
+		const { getByTestId } = render( <SegmentGroup { ...props } /> );
+		const notice1 = getByTestId( 'conflict-warning-1' );
+		const notice2 = getByTestId( 'conflict-warning-2' );
+
+		// Elements are in the document.
+		expect( notice1 ).toBeInTheDocument();
+		expect( notice2 ).toBeInTheDocument();
```

Comment:
Nit - in testing-library, all `getBy*` functions fail the test if element is not found, so technically checking for existence is redundant.

Reply:
Good deal, removed in 459751e7!